Multisig/UI/Assets/BalancesViewController/BalancesViewController.swift

```diff
@@ -52,6 +52,14 @@ class BalancesViewController: LoadableViewController, UITableViewDelegate, UITab
         }
 
         emptyView.setText("Balances will appear here")
+
+        NotificationCenter.default.addObserver(
+            self, selector: #selector(ownerKeyImported), name: .ownerKeyImported, object: nil)
+    }
+
+    @objc private func ownerKeyImported() {
```

Comment:
I believe after all discussions all what we need to do here is just dismiss the banner on any of its keys press, import/close.

Reply:
BalancesViewController created before SafeLoadedViewController is closed. So if the key was imported, we need to refresh it.